opentreemap/manage_treemap/views/management.py

```diff
@@ -13,15 +14,19 @@
 from django.utils.translation import ugettext as _
```

Comment:
The closed source version of this module imports `ugettext_lazy` for translating the `green_infrastructure_labels` content. I don't see any references to `green_infrastructure_labels` outside of this module, and it is only used once within this module, so it may be easier to just remove `green_infrastructure_labels` as a variable rather than import 2 versions of the translation function.